fem/fespace.hpp

```diff
@@ -587,6 +587,35 @@ class FiniteElementSpace
    bool Conforming() const { return mesh->Conforming() && cP == NULL; }
    bool Nonconforming() const { return mesh->Nonconforming() || cP != NULL; }
 
+   /** Set the prolongation operator of the space to an arbitrary sparse matrix,
+       creating a copy of the argument. */
+   void SetProlongation(const SparseMatrix* p)
+   {
+      if ( ! cP )
```

Comment:
Style suggestion: remove unnecessary spaces.

Reply:
I've removed these spaces in the version moved to the source file.